lib/console.rb

```diff
@@ -4,6 +4,8 @@ def initialize(output = $stdout, input = $stdin)
     @input = input
   end
 
+  public
```

Comment:
I don't think that you need to use the `public` keyword, by default every method is public I think...

So it is important to specify `private` and `protected`, but otherwise you can leave it as is